frb/data/analysis/CIGALE/VLT_HAWKI_J.dat

```diff
@@ -0,0 +1,2504 @@
+1032.966 -0.003623719
+1034.964 -0.001926364
+1036.962 -0.008814277
+1038.96 -0.008568594
+1040.958 -0.004413655
+1042.956 -0.00535324
+1044.954 -0.004430437
+1046.952 -0.004378619
+1048.95 -0.004220984
+1050.948 -0.005580146
+1052.946 -0.006125
+1054.944 -0.003383195
+1056.942 -0.003982434
+1058.94 -0.002864897
+1060.938 -0.009488896
+1062.936 -0.015799711
+1064.934 0.005307586
+1066.932 -0.006971801
+1068.93 -0.006866884
+1070.928 -0.005765643
+1072.926 -0.005004789
+1074.924 -0.008611511
+1076.922 -0.008227074
+1078.92 -0.00367977
+1080.918 -0.004459917
+1082.916 -0.01148873
+1084.914 -0.005380073
+1086.912 -0.005777478
+1088.91 -0.01156302
+1090.908 0.002850645
+1092.906 -0.001608606
+1094.904 0.000847113
+1096.902 -0.004717123
+1098.9 -0.002966294
+1100.898 -0.004120213
+1102.896 0.001047363
+1104.894 -0.003677554
+1106.892 -0.005906091
+1108.89 -0.004584253
+1110.888 -0.001069418
+1112.886 0.002235915
+1114.884 0.001116936
+1116.882 -0.00366399
+1118.88 0.002265912
+1120.878 -0.005922523
+1122.876 0.001641174
+1124.874 0.007355487
+1126.872 0.015800426
+1128.87 0.018320568
+1130.868 0.027420776
+1132.866 0.071776941
+1134.864 0.083741829
+1136.862 0.102686659
+1138.86 0.12002492
+1140.858 0.125748962
+1142.856 0.136439592
+1144.854 0.153737441
+1146.852 0.169855505
+1148.85 0.208103046
+1150.848 0.249182135
+1152.846 0.335922986
+1154.844 0.462093711
+1156.842 0.669638157
+1158.84 1.018598437
+1160.838 1.635083914
+1162.836 2.657725096
+1164.834 4.33525753
+1166.832 7.008089066
+1168.83 10.612573624
+1170.828 14.528684616
+1172.826 18.631008148
+1174.824 22.853862762
+1176.822 28.002552032
+1178.82 34.878364563
+1180.818 43.812431335
+1182.816 54.548770905
+1184.814 66.008293152
+1186.812 76.059898376
+1188.81 82.383026123
+1190.808 84.990081787
+1192.806 85.623893738
+1194.804 85.397277832
+1196.802 85.375801086
+1198.8 85.395812988
+1200.798 85.796058655
+1202.796 85.904945374
+1204.794 85.906478882
+1206.792 85.781234741
+1208.79 85.45980072
+1210.788 85.370269775
+1212.786 85.26234436
+1214.784 85.387191772
+1216.782 85.60546875
+1218.78 85.860671997
+1220.778 86.2290802
+1222.776 86.302246094
+1224.774 86.535797119
+1226.772 86.54800415
+1228.77 86.540176392
+1230.768 86.646110535
+1232.766 86.619354248
+1234.764 86.830291748
+1236.762 86.915473938
+1238.76 87.134895325
+1240.758 87.348274231
+1242.756 87.48538208
+1244.754 87.748947144
+1246.752 87.819061279
+1248.75 88.049118042
+1250.748 88.164352417
+1252.746 88.271530151
+1254.744 88.442085266
+1256.742 88.432418823
+1258.74 88.594039917
+1260.738 88.485527039
+1262.736 88.501808167
+1264.734 88.464630127
+1266.732 88.344520569
+1268.73 88.419898987
+1270.728 88.358894348
+1272.726 88.485702515
+1274.724 88.615234375
+1276.722 88.77722168
+1278.72 88.958572388
+1280.718 89.020973206
+1282.716 89.18964386
+1284.714 89.107505798
+1286.712 89.090499878
+1288.71 89.055152893
+1290.708 88.857528687
+1292.706 88.836494446
+1294.704 88.7603302
+1296.702 88.797012329
```

Comment:
Something seems amiss here.

Either these are microns, or there is (very!) high
transmission through in the UV through this filter.

Even if the latter is true, we shouldn't include this
portion of wavelength space.

Reply:
The filter was directly from ESO, so it seems that UV bleed is real. I've ensured that these are in angstroms. In any case, I just updated the J-band to remove everything blueward of 5000A, so that portion is now gone.

Reply:
Got it.  I expect they are right but thanks for removing.
  I worry the way we calculate extinction would have included
  that "bleeding", but would have to check to be sure.